app/connectors_service/connectors/cli/connector.py

```diff
@@ -6,6 +6,9 @@
 import asyncio
 from collections import OrderedDict
 
+from connectors_sdk.source import get_source_klass
```

Comment:
Looking at where `get_source_klass` and `get_source_klasses` are used, to me they belong to the connectors service app rather to the sdk?